legend-engine-xts-data-space/legend-engine-xt-data-space-protocol/src/main/java/org/finos/legend/engine/protocol/pure/v1/model/packageableElement/dataSpace/DataSpaceTemplateExecutable.java

```diff
@@ -14,11 +14,9 @@
 
 package org.finos.legend.engine.protocol.pure.v1.model.packageableElement.dataSpace;
 
-import org.finos.legend.engine.protocol.pure.v1.model.valueSpecification.raw.Lambda;
-
 public class DataSpaceTemplateExecutable extends DataSpaceExecutable
 {
```

Comment:
Could we instead do something like this. Don't like the idea combining classes like Lambda/PackageablePointer. 
For the below we can make the inline class the default implementation so it is backward compatible. 
````
public abstract class DataSpaceTemplateExecutable extends DataSpaceExecutable
{
    public String id;
    public String executionContextKey;
}

public class DataSpaceInlineTemplateExecutable extends DataSpaceTemplateExecutable 
{
    public Lambda query;
}

public class DataSpaceTemplateExecutablePointer extends DataSpaceTemplateExecutable 
{
    public PackageableElementPointer query;
}
````